apps/api/controllers/rooms/show.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module Controllers
+    module Rooms
+      class Show
+        include Api::Action
+
+        expose :room
+
+        def initialize(interactor: Interactors::Rooms::FetchRoom.new)
+          @interactor = interactor
+        end
+
+        def call(params)
+          room = @interactor.call(params[:id]).room
```

Comment:
Can we think of a more descriptive name than `interactor`?

Reply:
@jgayfer sure can! updated.